osbuild/pipeline.py

```diff
@@ -17,6 +17,29 @@
 from .util import osrelease
 
 
+DEFAULT_CAPABILITIES = {
+    "CAP_AUDIT_WRITE",
+    "CAP_CHOWN",
+    "CAP_DAC_OVERRIDE",
+    "CAP_DAC_READ_SEARCH",
+    "CAP_FOWNER",
+    "CAP_FSETID",
+    "CAP_IPC_LOCK",
+    "CAP_LINUX_IMMUTABLE",
+    "CAP_MAC_OVERRIDE",
+    "CAP_MKNOD",
+    "CAP_NET_BIND_SERVICE",
+    "CAP_SETFCAP",
+    "CAP_SETGID",
+    "CAP_SETPCAP",
+    "CAP_SETUID",
+    "CAP_SYS_ADMIN",
```

Comment:
These are the capabilities granted to `org.osbuild.rpm`, right? I don't know the full implications for each of them, but as this is the stage we are the most nervous about, would it make sense to give it an explicit list of capabilities, and maybe with some comments whey they are needed/fine?

Reply:
To all of the stages, but yes `org.osbuild.rpm` as well. I was mostly concerned about _not_ giving it `CAP_MAC_ADMIN` since this is the SELinux hole we punch.

Reply:
Agreed, that makes sense. Let's land this now and follow up with further hardening if we can in the future.